### DISCUSSION TITLE
Google Photos Albums connected via Jetpack does not work anymore

### DISCUSSION THREAD

**Discussion by wiag** [Automattic/jetpack#35007] [Q&A]
I've added Jetpack to my Wordpress blog and connected a Google Photo's account. Until a few weeks ago you could use the album view to select photos to upload to your blog, but recently this stopped working. When you select Albums from the view dropdown, the modal/lightbox disappears. I've disconnected and reconnected, and used another browser instead of Safari like Chrome and Firefox, to no avail.

**Comment by jeherve** [comment]
Picking from a Google Photo album seems to work for me right now. Could you check for Javascript errors in the block editor as you navigate the media modal, as explained here:
https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/

Let me know what you find!

**Comment by wiag** [comment]
Thanks for checking!

I’ve rebooted before I proceeded with the suggested steps, updated Wordpress again, removed the Google Photo’s connection again and reconnected. 

Google Chrome (Versie 120.0.6099.216 (Officiële build) (arm64)) 
now works (as before) with the following Console errors.

￼

Safari (Versie 17.2.1 (19617.1.17.11.12)) 
still does not, with the following console errors:

￼

… the difference being the resource index.mjs.map that fails to load. 

The file is indeed missing, I have no clue if this is the reason for it not working in Safari?
I’ve currently not turned on any of Jetpacks speed optimization options, but have in the past. 

Turning off content blocking and ITP does not make a difference. 

Note that in Safari picking photo’s from the library with most recent pictures first works as intended, but once you switch to Album view (which is the only way for us to find the necessary pictures in a big library) the modal disappears. 

Our community staff is all using Safari. 

Hope this helps. 

Kind regards,

Michiel

**Comment by jeherve** [comment]
I do not see the screenshots I'm afraid. Could you try adding them again?

> the difference being the resource index.mjs.map that fails to load.
> The file is indeed missing, I have no clue if this is the reason for it not working in Safari?

That's alright, .map files are not stricly necessary, they do not change the experience in the editor.

> once you switch to Album view (which is the only way for us to find the necessary pictures in a big library) the modal disappears.

Could you check the JavaScript console when that happens, see if a new error pops up?

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
